packages/build/tests/plugins/tests.js

```diff
@@ -736,6 +736,20 @@ test.serial('Plugins can specify multiple matching compatibility conditions', as
   })
 })
 
+test.serial('Plugins can specify non-matching compatibility.siteDependencies range', async (t) => {
+  await removeDir(`${FIXTURES_DIR}/plugins_compat_site_dependencies_range/.netlify`)
+  // We need this since the code dynamically requires the next dependency
+  await cpy(`${FIXTURES_DIR}/plugins_compat_site_dependencies_range/node_modules/next`, './node_modules/next', {
```

Comment:
This is ugly but works.
We can also setup a whole new temp directory with the expected hierarchy of files

Reply:
Could we do this instead?
This would be faster and more predictable.